api/v1beta1/types.go

```diff
@@ -144,6 +144,11 @@ type Subnets []SubnetSpec
 // +listMapKey=service
 type ServiceEndpoints []ServiceEndpointSpec
 
+// PrivateEndpoints is a slice of string.
```

Comment:
Isn't this a slice of a struct?

Reply:
It is a slice of PrivateEndpointSpec. Adjusted